### PR TITLE
changes-PluggableTextMorph-issue-22833

### PR DESCRIPTION
**Pull request by plantec**
changes related to PluggableTextMorph cleaning, issue Pharo 22833

### CODE REVIEW

**Comment by jecisc** [comment]
Hi,

PR should be against dev-0.1 for Iceberg.

**Comment by estebanlm** [comment]
against dev-1.0 you mean :)
I changed it.

**Comment by estebanlm** [comment]
... and fixed conflict.

**Comment by estebanlm** [comment]
... and changed the title to something a bit more understandable :)

File: Iceberg-TipUI.package/IceTipLocationPresenter.class/instance/initializeWidgets.st [hunk 1]
```diff
@@ -4,7 +4,7 @@ initializeWidgets
 	locationInput := self newTextInput autoAccept: true.
 	chooseButton := self newButton icon: self icon.
 	
-	locationInput whenBuiltDo: [ :w | w widget wrapFlag: false ]. 
+	locationInput whenBuiltDo: [ :w | "w widget beNotWrapped" ].
```
Comment (guillep) [inline Iceberg-TipUI.package/IceTipLocationPresenter.class/instance/initializeWidgets.st]: Is this a good change?

File: Iceberg-Plugin-Pharo.package/IcePharoNewBranchFromIssuePanel.class/instance/initializeWidgetsContents.st [hunk 2]
```diff
@@ -14,6 +14,5 @@ initializeWidgetsContents
 	issueNumberText whenBuiltDo: [ :w | w widget wrapFlag: false ].
 	issueText
 		whenBuiltDo: [ :w | 
-			w widget wrapFlag: false.
```
Comment (guillep) [inline Iceberg-Plugin-Pharo.package/IcePharoNewBranchFromIssuePanel.class/instance/initializeWidgetsContents.st]: Is this a good change?

**Comment by guillep** [comment]
This was originally done in filetree. We have moved the repository to Tonel some months ago already. I think we can close it and then check when we move more stuff to the new Spec version.

**Pull request closed without merging**